python/cuproj/pyproject.toml

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "cmake>=3.26.4",
+    "cython>=0.29,<0.30",
+    "ninja",
+    "rmm==23.8.*",
+    "scikit-build>=0.13.1",
+    "setuptools",
+    "wheel",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+
+[project]
+name = "cuproj"
+version = "23.8.0"
+description = "cuProj: GPU-Accelerated Coordinate Projection"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license = { text = "Apache 2.0" }
+requires-python = ">=3.9"
+dependencies = [
+    "cupy-cuda11x>=12.0.0",
```

Comment:
Yes, this is the cupy dependency I mentioned above. The test wheel script shouldn’t need to preinstall cupy from a source distribution, this lists the prebuilt wheels of cupy as a dependency.

Reply:
But it didn't work with just this.